packages/docs/src/_nav.md

```diff
@@ -0,0 +1,8 @@
+<!--
```

Comment:
This should be eventually moved out of this package (docs) since we want this package to be used as the theme for other projects (so it shouldn't know about datetime or table, for example)